server/admin_management/api/app/main.py

```diff
@@ -80,11 +80,6 @@ def custom_generate_unique_id(route: APIRouter):
     prefix=apiPrefix + "/application_admins",
     tags=["FAM Application Admin"],
 )
-app.include_router(
-    router_application.router,
-    prefix=apiPrefix + "/applications",
```

Comment:
removed the applications router, as we no longer use it anymore. Previously we use this endpoint to get all the applications for FAM admin (when grant app admin), now this information is included in the return of that get my access api